### ISSUE TITLE
[Feature]: Dark Mode

### ISSUE THREAD

**Issue by CStelian** [Elgg/Elgg#14284]
### Describe the feature

There should be an option for users inside their profile menu to turn on dark mode for the website. Store the theme in local storage so the preferred theme is always on.

### Is your feature request related to a problem?

No.

### Describe alternatives you've considered

Plugins. But most are unsupported.

**Comment by jdalsem** [comment]
I agree with your wish. We would need to update the theme to support darkmode. The preference could be a user site setting, but also it could respect the browser / OS preference (by default).

**Comment by jdalsem** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
